src/plugins/intel_gpu/tests/test_cases/tile_gpu_test.cpp

```diff
@@ -243,3 +243,427 @@ TEST(tile_gpu, basic_in1x2x2x2_axis_z) {
+                                 }),
+                    tile::along_b,
+                    2,
+            },
+            {
+                    tensor(1, 2, 2, 2),
+                    getValues<T>({
+                                   1.f, 0.f,
+                                   5.f, 1.5f,
+
+                                   2.f, 0.f,
+                                   6.f, 5.2f
+                                 }),
+                    tensor(1, 2, 2, 2),
+                    getValues<T>({
+                                   1.f, 0.f,
+                                   5.f, 1.5f,
+
+                                   2.f, 0.f,
+                                   6.f, 5.2f,
+                                 }),
+                    tile::along_f,
+                    2,
+            },
+            {
+                    tensor(1, 2, 2, 2),
+                    getValues<T>({
+                                   1.f, 0.f,
+                                   5.f, 1.5f,
+
+                                   2.f, 0.f,
+                                   6.f, 5.2f
+                                 }),
+                    tensor(1, 2, 2, 4),
+                    getValues<T>({
+                                   1.f, 0.f, 5.f, 1.5f,
+                                        1.f, 0.f, 5.f, 1.5f,
+
+                                   2.f, 0.f, 6.f, 5.2f,
+                                        2.f, 0.f, 6.f, 5.2f,
+                                 }),
+                    tile::along_y,
+                    2,
+            },
+            {
+                    tensor(1, 2, 2, 2),
+                    getValues<T>({
+                                   1.f, 0.f,
+                                   5.f, 1.5f,
+
+                                   2.f, 0.f,
+                                   6.f, 5.2f
+                                 }),
+                    tensor(1, 2, 4, 2),
+                    getValues<T>({
+                                   1.f, 0.f,
+                                        1.f, 0.f,
+                                   5.f, 1.5f,
+                                        5.f, 1.5f,
+
+                                   2.f, 0.f,
+                                        2.f, 0.f,
+                                   6.f, 5.2f,
+                                        6.f, 5.2f,
+                                 }),
+                    tile::along_x,
+                    2,
+            },
+            {
+                    tensor(1, 2, 1, 2),
+                    getValues<T>({1.f, 0.f, 5.f, 1.5f}),
+                    tensor(1, 2, 4, 2),
+                    getValues<T>({
+                                   1.f, 1.f, 1.f, 1.f,
+
+                                   0.f, 0.f, 0.f, 0.f,
+
+                                   5.f, 5.f, 5.f, 5.f,
+
+                                   1.5f, 1.5f, 1.5f, 1.5f,
+                                 }),
+                    tile::along_x,
```

Comment:
have you used clang-format for this file?

Reply:
Done